components/edit-collective/Form.js

```diff
@@ -84,14 +84,6 @@ class EditCollectiveForm extends React.Component {
 
     const { collective } = this.state;
 
-    this.showEditTiers = [COLLECTIVE, EVENT].includes(collective.type);
-    this.showExpenses = collective.type === COLLECTIVE || collective.isHost;
-    this.showEditGoals = collective.type === COLLECTIVE;
-    this.showHost = collective.type === COLLECTIVE;
-    this.defaultTierType = collective.type === EVENT ? 'TICKET' : 'TIER';
-    this.showEditMembers = [COLLECTIVE, ORGANIZATION].includes(collective.type);
-    this.showPaymentMethods = [USER, ORGANIZATION].includes(collective.type);
-
```

Comment:
@Betree : These are never used anywhere so I took the liberty of removing these. Let me know if you think otherwise.

Reply:
Good call :+1: